src/WpfMath/HorizontalBox.cs

```diff
@@ -1,4 +1,4 @@
-using System;
+﻿using System;

```

Comment:
Need to check what's going on with that line. I've edited it with VS2017 with the default settings; probably it has changed the BOM in the file.